apps/emqx_management/test/emqx_auth_mnesia_migration_SUITE.erl

```diff
@@ -95,12 +110,20 @@ do_import(Config, V) ->
                      access = allow
                     }],
                  lists:sort(Records)),
-    ?assertMatch([#emqx_user{
-                     login = {Type, <<"emqx_c">>}
-                    }], ets:tab2list(emqx_user)),
+    ?assertMatch([_, _], ets:tab2list(emqx_user)),
+    ?assertMatch([_], ets:lookup(emqx_user, {Type, <<"emqx_c">>})),
     Req = #{clientid => <<"blah">>}
           #{Type => <<"emqx_c">>,
-            password => "emqx_p"
+            password => <<"emqx_p">>
            },
     ?assertMatch({stop, #{auth_result := success}},
                  emqx_auth_mnesia:check(Req, #{}, #{hash_type => sha256})).
+
+test_clientid_import() ->
+    [#emqx_user{password = _Pass}] = ets:lookup(emqx_user, {clientid, <<"emqx_clientid">>}),
+    %% Req = #{clientid => <<"emqx_clientid">>,
```

Comment:
TODO: import of clientid password hash salt is broken on some releases (because base64 encoding was omitted). I am investigating whether this is recoverable.

Reply:
P.S. The error is not recoverable, unfortunately.